Documentation/project-docs/triage.md

```diff
@@ -48,4 +48,25 @@ In particular, we strive to follow these guidelines:
 * **Do** close issues that represent work we don't think makes sense for the current or upcoming release
 * **Do** close issues that provide little or questionable value to APIs that are included with the .NET Framework
 * **Do** favor issues that do not require modifying types that are included with the .NET Framework.
-* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
\ No newline at end of file
+* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
+
+## Porting to .NET Framework
+
+### Constraints for shipping in the .NET Framework
+ 
+Each release of .NET Framework ships to over one billion machines word-wide and installs as an *in-place* update, requiring each fix to meet a high level of quality and compatibility.  Each fix is reviewed extensively to determine which release if any is most appropriate.
```

Comment:
I now what you mean by we don't plan to provide another side-by-side release, so for all intents and purposes, the .NET Framework will be in-place only moving forward (4.0, 4.5, 4.5.1, 4.6, 4.6.1 etc).